services/user.py

```diff
@@ -0,0 +1,47 @@
+from django.contrib.auth import get_user_model
+from django.db import transaction
+
+from db.models import User
+
+
+def create_user(username: str,
+                password: str,
+                email: str | None = None,
+                first_name: str = "",
+                last_name: str = ""
+                ) -> User:

```

Comment:
Place each argument on a new line so that there would be no arguments on the same line with opening and closing parenthesis. It will help to improve the code style and readability